### ISSUE TITLE
[FEATURE REQUEST] Thunderbird Support

### ISSUE THREAD

**Issue by cfouche3005** [largestgithubuseronearth/addwater#19]
Do you plan to add support for the [thunderbird gnome theme](https://github.com/rafaelmardojai/thunderbird-gnome-theme) by the same dev.
Looking at the install script, it's nearly the same logic, as thunderbird use firefox as his backend, but different places.
Nonetheless, thanks for your amazing app.

**Comment by largestgithubuseronearth** [comment]
I've always kept the door open for Thunderbird support because you're right that the installation process is nearly the same. However, my code for downloading theme updates wouldn't carry over: I rely on the dev putting out full, versioned releases for the Firefox Theme. This makes it easy to see when a new update is available because you can just compare the tag to the currently installed version. 

However, since the Thunderbird Theme isn't completed, the dev doesn't make tagged releases. This means I would need to re-do most of the logic that handles updates. Not impossible, I designed the project to accommodate this in the future, but it would just take some time.

tl;dr I'm planning to support the Thunderbird Theme in the future but you won't see that soon. I'll leave this issue open for it though.
